src/map/status.cpp

```diff
@@ -676,14 +676,18 @@ uint64 EnchantgradeDatabase::parseBodyNode( const ryml::NodeRef& node ){
 				grade->refine = refine;
 			}
 
-			if( this->nodeExists( gradeNode, "Chance" ) ){
-				uint16 chance;
+			if( this->nodeExists( gradeNode, "Chances" ) ){
+				uint16 chance,refineIndex;
 
-				if( !this->asUInt16Rate( gradeNode, "Chance", chance ) ){
-					return 0;
+				for( const ryml::NodeRef& chanceNode : gradeNode["Chances"] ){
+					if( !this->asUInt16( chanceNode, "Refine", refineIndex ) ){
+						return 0;
+					}
+					if( !this->asUInt16( chanceNode, "Chance", chance ) ){
```

Comment:
```suggestion
					if( !this->asUInt16Rate( chanceNode, "Chance", chance ) ){
```
This keeps it between 0~10000.